llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -13658,6 +13658,7 @@ bool SITargetLowering::isCanonicalized(Register Reg, const MachineFunction &MF,
     case Intrinsic::amdgcn_frexp_mant:
     case Intrinsic::amdgcn_fdot2:
     case Intrinsic::amdgcn_trig_preop:
+    case Intrinsic::amdgcn_tanh:
```

Comment:
This isn't tested

Reply:
It was but not by this instruction, since this intrinsic is used for those different types. I'll remove them then?